test/parallel/test-http-parser.js

```diff
@@ -92,7 +91,7 @@ function expectBody(expected) {
 // Simple request test.
 //
 {
-  const request = Buffer.from(`GET /hello HTTP/1.1${CRLF}${CRLF}`);
+  const request = Buffer.from('GET /hello HTTP/1.1\r\n\r\n');
```

Comment:
This is a non whitespace change in a commit that only refers to changes in whitespace. Please also make note of this is the git message body.